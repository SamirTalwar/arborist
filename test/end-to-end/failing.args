test/end-to-end/failing/Main.elm
