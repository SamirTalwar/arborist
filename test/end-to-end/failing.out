[32msome tests pass PASSED
[0m[31mbut some tests fail FAILED
  Expected:
  False
  Actual:
  True
[0m